android/filament-utils-android/src/main/java/com/google/android/filament/utils/KtxLoader.kt

```diff
@@ -76,7 +76,14 @@ object KtxLoader {
         return Skybox(engine, nativeSkybox)
     }
 
+    fun getSphericalHarmonics(buffer: Buffer): FloatArray {
+        val floatBuffer = FloatBuffer.allocate(3 * 9)
```

Comment:
Why go through a `FloatBuffer`, why not use a regular array?

Reply:
Changed to `FloatArray`